src/openforms/emails/digest.py

```diff
@@ -179,3 +190,36 @@ def collect_broken_configurations() -> list[BrokenConfiguration]:
         )
 
     return broken_configurations
+
+
+def collect_invalid_certificates() -> list[InvalidCertificate]:
+    today = timezone.now()
+
+    invalid_certs = []
+    for cert in Certificate.objects.all():
+        time_until_expiry = cert.expiry_date - today
+
+        if time_until_expiry <= timedelta(days=14):
+            invalid_certs.append(
+                InvalidCertificate(label=cert.label, expiry_date=cert.expiry_date)
+            )
+
+        if not (is_valid := cert.is_valid_key_pair()):
+            invalid_certs.append(
+                InvalidCertificate(label=cert.label, is_valid_pair=is_valid)
+            )
```

Comment:
what about when a certificate is both close to expiry, and it's also not valid pair? You'd be reporting the same certificate twice then :)

Probably better to do something like:

```
Found invalid certificates:

    Certificate A: will expire soon, invalid keypair
    Certificate B: will expire soon
```